src/app/root-store/server-data/user-tags/user-tags.reducer.ts

```diff
@@ -83,3 +84,10 @@ export function reducer(state = initialState, action: UserTagsActionsUnion): Use
     }
   }
 }
+
+export const getUserTagsState = createFeatureSelector<UserTagsState>('userTags');
```

Comment:
Moved getState and isLoading to `.reducer.ts` for consistency, because everywhere else these are placed in `.reducer.ts`.